install.sh

```diff
@@ -115,6 +115,16 @@ rosdep -q update
 echo -e "\e[32mDone: Configuring rosdep\e[0m"
 echo ""
 
+
+echo -e "\e[94mSetting up groups\e[0m"
+sudo addgroup flirimaging
+sudo usermod -a -G flirimaging administrator
```

Comment:
This script is meant to be generic for users even if they configure things slightly differently so can we use `${USER}` instead of administrator?

Reply:
Updated.